specification/ekg/resource-manager/readme.md

```diff
@@ -0,0 +1,91 @@
+# EkgService
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for EkgService.
+
+
+
+---
+
+### Java multi-api
+
+``` yaml $(java) && $(multiapi)
+batch:
+  - tag: package-2018-12-03
+```
+
+### Tag: package-2018-12-03 and java
+
+These settings apply only when `--tag=package-2018-12-03 --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-2018-12-03' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.ekgservice.v2018-12-03
+  output-folder: $(azure-libraries-for-java-folder)/ekgservice/resource-manager/v2018-12-03
```

Comment:
According to the error [here](https://github.com/Azure/azure-rest-api-specs/pull/4898/checks?check_run_id=44453942) :
```
AutoRest code generation utility [version: 2.0.4283; node: v8.12.0]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/root/.autorest/@microsoft.azure_autorest-core@2.0.4289/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4289)
   Loading AutoRest extension '@microsoft.azure/autorest.go' (~2.1.127->2.1.127)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.38->2.3.38)
FATAL: System.InvalidOperationException: codegen for preview swagger Microsoft.EKG/preview/2018-12-03/ekgSwagger.json must be under a preview subdirectory
```

The output folder should contain preview in it.

consider implementing a separate file for go. Example [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cost-management/resource-manager#go)

Here is the go [entry for a preview spec](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cost-management/resource-manager/readme.go.md#tag-package-2018-08-preview-and-go).

Reply:
Changes have been made